docs/_includes/common/filter-menu.html

```diff
@@ -0,0 +1,541 @@
+<div id="filter-menu">
+    <h2><span class="secondary-text">@ebay/skin/</span>filter-menu</h2>
+
+    <h3 id="filter-menu-aria">Filter Menu – ARIA Version – Default</h3>
+    <p>The ARIA version of the filter menu is a <em>statelful</em> menu. It's primary purpose is used in conjunction with <a href="#filter-menu-button">filter-menu-button</a>,
+        but can alternatively be used as a standalone menu when necessary.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="filter-menu-aria">
+                <form name="filter-menu-aria-1">
+                    <div class="filter-menu-aria__items" role="menu">
+                        <div class="filter-menu-aria__item" role="menuitemcheckbox" aria-checked="false">
+                            <span class="filter-menu-aria__status" hidden>
+                                <svg class="icon icon--checkbox-unchecked" focusable="false" height="18" width="18">
```

Comment:
Is background icon supported? Would hugely simplify the markup down to a single span for those users who have the simple "out of the box" use case.

Reply:
Do we want more background SVG? In the page? My vision is that the core component would leverage `<ebay-icon>`, and the toggle would be easy.

And, the easy of "markup" is at the expense of the page weight, IMO.

Reply:
I'm not talking about the core component though. CoreUI is already 100% foreground SVG. We have to remember there are teams that use Skin only though. Background SVG makes life much easier for them (teams specifically asked us for this). Especially in this case. Yes, the page weight issue does need addressing at some point if we continue our support for background svg, but we aren't adding any extra weight (i.e. new icons) here.

Reply:
Fine by me. I know it's a bit of a hassle supporting both foreground and background, but I think we should continue it for now as one of our "developer experience" values. When I fix the test pages next week I'll add test cases for background and foreground for all of our modules. I'm thinking we can make use of the HTML5 details element on the main website too to tuck away the foreground SVG versions.

Reply:
I've made some changes here, so please check when you get a moment.